js/utilities.js

```diff
@@ -2251,3 +2251,18 @@
         UPDATE: "UPDT",   //update
         READ: "READ"        //read
       });
+
+    module._errorStatus = Object.freeze({
+      forbidden : "Forbidden",
+      itemNotFound : "Item Not Found",
+      facetingError: "Faceting Error",
```

Comment:
Please change this to "Invalid Facet Filters",  so it would be in the same format as others.